BitwardenShared/UI/Vault/Vault/VaultItemSelection/VaultItemSelectionState.swift

```diff
@@ -42,6 +42,11 @@ struct VaultItemSelectionState: Equatable {
 
     /// The search term used to find ciphers that match the OTP key.
     var ciphersMatchingName: String? {
-        otpAuthModel.issuer ?? otpAuthModel.accountName
+        switch totpKeyModel.totpKey {
+        case let .otpAuthUri(otpAuthModel):
+            return otpAuthModel.issuer ?? otpAuthModel.accountName
+        case .standard, .steamUri:
+            return nil
```

Comment:
🤔 Is it ok that no vault items are streamed for standard and steam Uri? I would guess so but wanted to ask just in case.

Reply:
I think that might be the only option. This string is just used to filter the list down to items we think match. But we might only have `steam://JBSWY3DPEHPK3PXP` or just `JBSWY3DPEHPK3PXP` in these cases, which makes it hard for us to guess as to what matches. 

But UI-wise, we already handle this because the older OTPAuth version could have the same problem (e.g. on a url that only passes secret with no issuer, etc). In fact, that's why this is an optional string. So you can always search for the item you want to add this one to and you'll still find it. 

One other note is that due to the way the Authenticator app saves these codes, I think it will actually generate an otpauth url style code for Standard codes manually entered. So let's say I put "Amazon" as the name and `JBSWY3DPEHPK3PXP`, when it passes it over to the PM app, it will likely hand that off in such a way that "Amazon" ends up as the issuer and is used here. 👍

Reply:
Roger makes sense, thanks!